test/test_decimal128.cpp

```diff
@@ -182,6 +182,9 @@ TEST(Decimal128_Aggregates)
                 sum += val;
                 count++;
             }
+            else {
+                obj.set_null(col_dec);
```

Comment:
Shouldn't the default value be null for nullable columns? Is this a no-op?

Reply:
Yes it is the default - I changed this to a check that it is null. This was copied from other test code.